addon/run-task.js

```diff
@@ -215,25 +215,32 @@ export function throttleTask(obj, name, timeout = 0) {
    ```
 
    @method cancelTask
+   @param { Object } obj the entwined object that was provided with the original runTask or scheduleTask call
```

Comment:
Can we use the word `entangled` here instead? This would match the wording used in the README.